sources/packages/backend/apps/api/src/route-controllers/education-program/_tests_/e2e/education-program.institutions.getEducationProgram.e2e-spec.ts

```diff
@@ -0,0 +1,150 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
```

Comment:
Please follow the file naming conventions for E2E test file. In this case, it must include `controller`